src/components/Login/index.js

```diff
@@ -21,7 +21,7 @@ const Login = () => {
         if (token) {
           dispatch(loginAction({ token }));
           window.localStorage.setItem("token", token);
```

Comment:
it should get stored in reducer as well from here. check from where we are currently stoeing it?

Reply:
already implemented